### PR TITLE
allow pickling xdr/dcd trajectory readers

### PR DESCRIPTION
**Pull request by kain88-de**
Changes made in this Pull Request:
 Add methods to pickle the low level xdr readers. This allows us to write a version of pmda that is independent of the dask scheduler.

PR Checklist
------------
 - [x] Tests?
 - [x] Docs?
 - [x] CHANGELOG updated?
 - [ ]~ Issue raised/referenced?~

### CODE REVIEW

**Comment by kain88-de** [comment]
I'll check that I add this to libdcd as well

**Comment by kain88-de** [comment]
libdcd work as well.

File: testsuite/MDAnalysisTests/formats/test_libdcd.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
 from six.moves import zip
 
 from collections import namedtuple
+import pickle
```
Comment (orbeckst) [inline testsuite/MDAnalysisTests/formats/test_libdcd.py]: Shouldn't this be using pickle from `six`?

Comment (kain88-de) [reply testsuite/MDAnalysisTests/formats/test_libdcd.py]: good catch. Haven't thought about it.

**Pull request merged**